packages/swapper/src/swappers/ChainflipSwapper/swapperApi/getTradeRate.ts

```diff
@@ -1,11 +1,19 @@
-import type { GetTradeRateInput, SwapperDeps, TradeRateResult } from '../../../types'
-import { getRateOrQuote } from '../utils/getRateOrQuote'
+import type { GetTradeRateInput, SwapperDeps, TradeRateResult, TradeRateStep } from '../../../types'
+import { getQuoteOrRate } from '../utils/getQuoteOrRate'
 
 export const getTradeRate = async (
   input: GetTradeRateInput,
   deps: SwapperDeps,
 ): Promise<TradeRateResult> => {
-  // A quote is a rate with guaranteed BIP44 params (account number/sender),
-  // so we can return quotes which can be used as rates, but not the other way around
```

Comment:
Not applicable anymore, both trade quotes and trade rates *may* (in the case of trade rates, only with wallet connected) have BIP44ish params